client/Cargo.toml

```diff
@@ -32,7 +32,7 @@ quinn-proto = "0.8.3"
 rand = "0.7.0"
 rand_chacha = "0.2.2"
 rayon = "1.5.3"
-reqwest = { version = "0.11.10", default-features = false, features = ["blocking", "rustls-tls", "json"] }
+reqwest = { version = "0.11.10", default-features = false, features = ["blocking", "brotli", "deflate", "gzip", "rustls-tls", "json"] }
```

Comment:
No code changes to `HttpSender` necessary; enabling these features turns them on in `reqwest` by default.

Reply:
heh nice